theme/src/ts/misc.ts

```diff
@@ -90,7 +90,7 @@ function bindToggles(selector) {
     });
 }
 
-function generateOnThisPage() {
+export function generateOnThisPage() {
```

Comment:
I had to export this, and then redraw the sidebar after the experiment was engaged, as menu items change.